docs/content/connection-options.md

```diff
@@ -183,6 +183,11 @@ These are the other options that MySqlConnector supports.  They are set to sensi
     <td>True to have MySqlDataReader.GetValue() and MySqlDataReader.GetDateTime() return DateTime.MinValue for date or datetime columns that have disallowed values.</td>
   </tr>
   <tr>
+    <td>Default Command Timeout, Command Timeout, DefaultCommandTimeout</td>
+    <td>0</td>
```

Comment:
Isn't the default `30`?

Reply:
Good catch; this should have been fixed in 2fef1049a2f11cde2200dcb554b454fdf00ed4d5.